crates/client/mapping-sync/src/sync_blocks.rs

```diff
@@ -68,7 +68,7 @@ where
     let block = client.runtime_api().current_block(substrate_block_hash).map_err(|e| format!("{:?}", e))?;
     let block_hash = block.header().hash(PedersenHasher::default());
     let mapping_commitment =
-        mc_db::MappingCommitment::<B> { block_hash: substrate_block_hash, starknet_block_hash: block_hash };
+        mc_db::MappingCommitment::<B> { block_hash: substrate_block_hash, starknet_block_hash: block_hash.into() };
```

Comment:
not related to _this_ change, but are we sure that we want to define the variables such that we have

```
{
    block_hash: substrate_block_hash,
    starknet_block_hash: block_hash.into()
}
```

and not 

```
{
    block_hash,
    starknet_block_hash: starknet_block_hash.into()
}
```

Reply:
Yes, cf [frontier](https://github.com/paritytech/frontier/blob/master/client/mapping-sync/src/lib.rs#L61)

Reply:
To have what ? Well here it's quite important we don't differ much from an audited and proven secure codebase that frontier is.

Reply:
@tdelabro has worked the most on this part and could help clarify there.

Reply:
my comment was just about naming though